### ISSUE TITLE
Focus indicator on video control

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/gatsby-theme-carbon#1398]
On the new [accessibility page for slider](https://carbondesignsystem.com/components/slider/accessibility) the short video in the Labels and updates section can be operated by keyboard; however, there is no focus indicator on the play button. It takes focus, but there looks like there is some suppression of the focus indicator.
